test/tools/ossfuzz/protoToYul.h

```diff
@@ -27,30 +29,40 @@ namespace test
 {
 namespace yul_fuzzer
 {
-class Function;
-
-std::string functionToString(Function const& input);
-std::string protoToYul(uint8_t const* data, size_t size);
-std::string createHex(std::string const& _hexBytes);
-std::string createAlphaNum(std::string const& _strBytes);
-std::ostream& operator<<(std::ostream& _os, BinaryOp const& _x);
-std::ostream& operator<<(std::ostream& _os, Block const& _x);
-std::ostream& operator<<(std::ostream& _os, Literal const& _x);
-std::ostream& operator<<(std::ostream& _os, VarRef const& _x);
-std::ostream& operator<<(std::ostream& _os, Expression const& _x);
-std::ostream& operator<<(std::ostream& _os, BinaryOp const& _x);
-std::ostream& operator<<(std::ostream& _os, VarDecl const& _x);
-std::ostream& operator<<(std::ostream& _os, TypedVarDecl const& _x);
-std::ostream& operator<<(std::ostream& _os, UnaryOp const& _x);
-std::ostream& operator<<(std::ostream& _os, AssignmentStatement const& _x);
-std::ostream& operator<<(std::ostream& _os, IfStmt const& _x);
-std::ostream& operator<<(std::ostream& _os, StoreFunc const& _x);
-std::ostream& operator<<(std::ostream& _os, Statement const& _x);
-std::ostream& operator<<(std::ostream& _os, Block const& _x);
-std::ostream& operator<<(std::ostream& _os, Function const& _x);
-std::ostream& operator<<(std::ostream& _os, ForStmt const& _x);
-std::ostream& operator<<(std::ostream& _os, CaseStmt const& _x);
-std::ostream& operator<<(std::ostream& _os, SwitchStmt const& _x);
-}
+class protoConverter
+{
+public:
+	protoConverter() {}
```

Comment:
Beter to not mention the constuctors and destructors at all.

Reply:
I will be using them shortly in an upcoming PR that adds scoping to variable declarations. Can we keep it until then, and take a call?